index.html

```diff
@@ -77,6 +77,9 @@
                     <li>
                         <a href="status">Status</a>
                     </li>
+                    <li>
+                        <a href="docs/orga/getting-in-touch.html">Contact</a>
+                    </li>
```

Comment:
this header needs to be added to the other html pages

Reply:
only places I found with this header was
* `feedstocks.html.tmpl`
* `index.html`
* `404.html`

Did I miss any?

Reply:
I think that is it. The rest of the pages are external links.